operator/pkg/ciliumendpointslice/endpointslice.go

```diff
@@ -225,8 +286,19 @@ func (c *Controller) runCiliumNodesUpdater(ctx context.Context) error {
 		c.logger.Warn("Couldn't get CiliumNodes store", logfields.Error, err)
 		return err
 	}
+
 	for event := range c.ciliumNodes.Events(ctx) {
+		switch event.Kind {
+		case resource.Upsert:
+			c.logger.Debug("Got Upsert CiliumNode event", logfields.NodeName, event.Key.String())
```

Comment:
Nit. `String()` should not be called on log arguments, as slog already does that internally only if the line is actually going to be emitted.